docs/sphinx/api/default_ops.rst

```diff
@@ -638,31 +638,67 @@ operations, each operating on 2 qubits.
 Photonic Operations on Qudits
 =============================
 
-These operations are valid only on the `photonics` target which does not support the quantum operations above.
+These operations are valid only on the `photonics` target which does not support
+the quantum operations above.
 
-:code:`plus`
+:code:`create`
```

Comment:
These names don't match the RFC here https://github.com/NVIDIA/cuda-quantum/issues/1937. 
@Omar-ORCA : Can you please add the proposal to RFC?

Reply:
I added the proposal as a comment